client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java

```diff
@@ -76,12 +77,17 @@ public void registerJob(JobShuffleContext context) {
     if (lifecycleManager == null) {
       synchronized (RemoteShuffleMaster.class) {
         if (lifecycleManager == null) {
-          celebornAppId = FlinkUtils.toCelebornAppId(lifecycleManagerTimestamp, jobID);
-          LOG.info("CelebornAppId: {}", celebornAppId);
+          String applicationId = FlinkUtils.toCelebornAppId(lifecycleManagerTimestamp, jobID);
```

Comment:
Please revert this change. Meanwhile, remove line 87~89.

Reply:
> Please revert this change. Meanwhile, remove line 87~89.

ok, let's revert it.
should we describe that the flag is only for spark and mr in doc ?